### PR TITLE
Stmt dump to sif

### PR DESCRIPTION
**Pull request by kkaris**
This PR updates how the Sif dump is made.

Instead of calling the DB directly, the Sif dumper now makes use of the indranet assembler's `make_df` method and the FullPaStmt dump.

The dump order is switched so that the Sif dumper can query for the FullPaStmt dump.

### CODE REVIEW

**Comment by kkaris** [comment]
I realized we also wanted to add residue and position in the sif dump, I'll need to post a separate PR there.

**Comment by kkaris** [comment]
See https://github.com/sorgerlab/indra/pull/1222

File: indra_db/managers/dump_manager.py [hunk 1]
```diff
@@ -458,6 +450,13 @@ def dump(principal_db, readonly_db, delete_existing=False, allow_continue=True,
         else:
             logger.info("Statement dump exists, skipping.")
 
+        if not allow_continue or not Sif.from_list(starter.manifest):
+            logger.info("Dumping sif from the readonly schema on principal.")
+            Sif(db=principal_db, date_stamp=starter.date_stamp)\
```
Comment (johnbachman) [inline indra_db/managers/dump_manager.py:455]: Is the db=principal_db actually required as an arg since the Sif dumper class doesn't use the DB anymore?

**Comment by kkaris** [comment]
I'm closing this one as I have now merged the wanted updates into another branch.

**Pull request closed without merging**